ddl/column_type_change_test.go

```diff
@@ -1718,3 +1722,53 @@ func (s *testColumnTypeChangeSuite) TestChangingAttributeOfColumnWithFK(c *C) {
 
 	tk.MustExec("drop table if exists orders, users")
 }
+
+// Close issue #23202
+func (s *testColumnTypeChangeSuite) TestDDLExitWhenCancelMeetPanic(c *C) {
```

Comment:
Could we set `tidb_ddl_error_count_limit` to a small value to make this test run faster？